code/game/objects/items/devices/defib.dm

```diff
@@ -262,10 +262,12 @@
 
 //Checks for various conditions to see if the mob is revivable
 /obj/item/weapon/shockpaddles/proc/can_defib(mob/living/carbon/human/H) //This is checked before doing the defib operation
-	if((H.species.flags & NO_SCAN))
+	if((H.species.flags & NO_SCAN &! H.species.reagent_tag & IS_VOX))
```

Comment:
This conditional is equivalent to:
`H.species.flags & NO_SCAN & (!h.species.reagent_tag) & IS_VOX`
Since `NO_SCAN` is 4 and `IS_VOX` IS 2, the bitwise and will always return 0. Basically, defibs can't fail this check, period, so they'll always try.
I recommend changing it to `H.species.flags & NO_SCAN && !(H.species.reagent_tag & IS_VOX)` to do what you want it to do. Refer to http://www.byond.com/docs/ref/info.html#/operator for order of operations.

Reply:
Ahh okay, thanks for the information. I'll go ahead and make the necessary change